galaxy_api/api/urls.py

```diff
@@ -11,4 +11,5 @@
     path("", views.ApiRootView.as_view(), name="root"),
     path("v3/", include(v3_urls, namespace="v3"), name="v3_root"),
     path("v3/_ui/", include(ui_urls, namespace="ui")),
+    path("api/", views.SlashApiRedirectView.as_view(), name="compat_redirect"),
```

Comment:
Please add a note explaining that this is temporary fix.

Reply:
Think this would be a permanent fix.

Reply:
Good idea.

Reply:
Updated. (force pushed to ammend and sign the commit)